saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -116,6 +116,12 @@ setup_tls() {
     echo "cert-tool exists on the fs (backward compatibility)"
     create_certificates_cert_tool
   fi
+
+  if [[ "$IS_CCM_V2_ENABLED" == "true" ]]; then
+    echo "CCMv2 is enabled while creating an environment so ssl client verification is turned off and localhost:9443 used"
+    sed -i "s/ssl_verify_client on;/ssl_verify_client off;/" /etc/nginx/sites-enabled/ssl-template
+    sed -i "s/listen       9443;/listen       localhost:9443;/" /etc/nginx/sites-enabled/ssl-template
```

Comment:
it might be better to use regexp for the spaces instead of the exact number of spaces. like if someone would delete a space in ssl-template it would break this and there is no test to validate this

Reply:
my comment was for both lines